prody/proteins/interactions.py

```diff
@@ -1098,7 +1114,12 @@ def showInteractionMap(atoms):
             raise TypeError('coords must be an object '
                             'with `getCoords` method')
 
-    InteractionsMap = calcInteractionMatrix(atoms)
+    try:
+        scoring = kwargs['scoring']
+    except:
+        scoring = [2, 3, -1, 3, 3, 1]  #HBs_calculations, SBs_calculations, SameChargeResidues, Pi_stacking, Pi_cation, Hydroph_calculations
```

Comment:
Same comment here

Reply:
Actually, we don't need the kwargs like this. We want to do it the other way round and provide them straight through to calcInteractionMatrix. 

You could delete these few lines as calcInteractionMatrix has them already and this function doesn't use them itself.